framework/src/main/java/org/checkerframework/framework/type/DefaultTypeHierarchy.java

```diff
@@ -209,80 +208,18 @@ protected String defaultErrorMessage(
      *     supertype for the current top.
      */
     protected boolean isPrimarySubtype(AnnotatedTypeMirror subtype, AnnotatedTypeMirror supertype) {
-        return isPrimarySubtype(subtype, supertype, false);
-    }
-
-    /**
-     * Compare the primary annotations of {@code subtype} and {@code supertype}.
-     *
-     * @param annosCanBeEmtpy indicates that annotations may be missing from the typemirror
-     * @return true if the primary annotation on subtype {@literal <:} primary annotation on
-     *     supertype for the current top or both annotations are null. False is returned if one
-     *     annotation is null and the other is not.
-     */
-    protected boolean isPrimarySubtype(
```

Comment:
Both this method and `isAnnoSubtype` either called with `annosCanBeEmtpy = false` or the code checked that the annotations were nonnull before calling them.  So I just deleted these methods.